src/test/java/org/fundacion/jala/converter/core/metadata/ExportTypeFileTest.java

```diff
@@ -0,0 +1,30 @@
+package org.fundacion.jala.converter.core.metadata;
```

Comment:
Is missing Copyright for this file

Reply:
We agreed that javadoc and copyright would not be put in the tests